app/src/main/kotlin/no/nav/tiltakspenger/vedtak/routes/behandling/BehandlingBenkRoutes.kt

```diff
@@ -42,6 +43,9 @@ fun Route.behandlingBenkRoutes(
                         else -> null
                     },
                     status = finnStatus(it),
+                    typeBehandling = if (it is Søknadsbehandling) "Førstegangsbehandling" else "Ukjent",
```

Comment:
Til senere hadde det sikkert vært nice å ha dette i en egen metode (og enum?) for å lettere kunne skalere senere :) Men det fungerer for nå